photon-lib/src/main/native/cpp/photonlib/PhotonTrackedTarget.cpp

```diff
@@ -77,7 +77,14 @@ Packet& operator<<(Packet& packet, const PhotonTrackedTarget& target) {
          << target.poseAmbiguity;
 
   for (int i = 0; i < 4; i++) {
-    packet << target.corners[i].first << target.corners[i].second;
+    packet << target.minAreaRectCorners[i].first
+           << target.minAreaRectCorners[i].second;
+  }
+
+  packet << static_cast<uint8_t>(target.detectedCorners.size());
```

Comment:
Is there anything that enforces `target.detectedCorners.size() < std::numeric_limits<unit8_t>::max()`? If not then it might be good to do a `std::min(target.detectedCorners.size(), std::numeric_limits<unit8_t>::max())` here just to be defensive.

Reply:
On the Photon side it's guaranteed, but yeah I can make it more defensive